.gitignore

```diff
@@ -1 +1,84 @@
-*.pyc
+# Developer env specific files
+.DS_Store
+.vscode
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
```

Comment:
This seems like the only applicable line. Should the _.gitignore_ be reduced to just this?

Reply:
More or less. I'm in the process of adding unit tests and would need some.

Please feel free to trim or ignore the pr as I can include a more focused gitignore once unit tests begin to me merged :)